python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_row_count_postprocessing.py

```diff
@@ -1,18 +1,46 @@
 import os
 from typing import Any, Dict, cast
 
+import pytest
 from dagster import (
     AssetExecutionContext,
     _check as check,
     materialize,
 )
 from dagster._core.definitions.metadata import IntMetadataValue
 from dagster_dbt.asset_decorator import dbt_assets
-from dagster_dbt.core.resources_v2 import DbtCliResource
+from dagster_dbt.core.resources_v2 import DbtCliInvocation, DbtCliResource
 
+from ..conftest import _create_dbt_invocation
 from ..dbt_projects import test_jaffle_shop_path
 
 
+@pytest.fixture(name="standalone_duckdb_dbfile_path")
+def standalone_duckdb_dbfile_path_fixture(request) -> None:
+    """Generate a unique duckdb dbfile path for certain tests which need
+    it, rather than using the default one-file-per-worker approach.
+    """
+    jaffle_shop_duckdb_db_file_name = "cool_jaffle_shop"
+    jaffle_shop_duckdb_dbfile_path = f"target/{jaffle_shop_duckdb_db_file_name}.duckdb"
```

Comment:
Should probably parameterize this by the test name to guarantee isolations between tests

https://stackoverflow.com/a/34732269

Reply:
had this initially but changed it to test something and forgot to change it back 😅 , thanks